src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  height: 408px;
```

Comment:
- Don't hardcode height(Don't fix container size (if there is no such a requirement))